models/topic-model.js

```diff
@@ -65,6 +65,30 @@ const fetchCommentsByArticle = (article_id) => {
     });
 };
 
+const fetchPostCommentByArticle = (username, body, article_ID) =>{
+  if(!username || !body){
```

Comment:
There are not null constraints on the seed, this should be handled by psql rather than a custom error